spec/factory_girl/attribute/dynamic_spec.rb

```diff
@@ -29,7 +29,22 @@
     let(:result) { "other attribute value" }
 
     before do
-      subject.stubs(attribute_defined_on_attribute: result)
+      RSpec.configure do |config|
+        config.mock_with :rspec do |mocks|
+          mocks.verify_partial_doubles = false
+        end
+      end
+      allow(subject).to(
+        receive(:attribute_defined_on_attribute).and_return result,
+      )
```

Comment:
looks I managed to break hound 😛  (this is actually valid ruby and `rspec` runs it happily